format/index.md

```diff
@@ -527,6 +529,28 @@ the value of each member of a `links` object is not constrained by the
 current version of the spec. It should be assumed that anything can be
 allowed in the future for such values: object, array, or scalar.
 
+### JSON API Object <a href="#document-structure-jsonapi-object" id="document-structure-jsonapi-object" class="headerlink"></a>
+
+A JSON API document **MAY** include information about its implementation under
+a top level `"jsonapi"` member. If present, it **MUST** be an object containing
+a `version` member whose value is a string indicating the highest JSON API
+version supported. Other than `"meta"`, servers **MUST NOT** include any
```

Comment:
I believe that's covered in my note below:
> Note: Because JSON API is committed to making additive changes only, the version string primarily indicates which new features a server may support.